src/core/ThatMark.ts

```diff
@@ -1,10 +1,10 @@
-import { SelectionWithEditor } from "../typings/Types";
+import { Target } from "../typings/target.types";
```

Comment:
Note that now the `ThatMark` class doesn't know anything about selections; felt specific to the `CommandRunner`